### PR TITLE
Add model/migration and handlers for form1299 

### PR DESCRIPTION
**Pull request by amitch23**
## Description:
This PR adds the model, migration, and model tests for Form1299. 

## Testing Steps:

1. Run the migration: `make db_dev_migrate`.
2. Verify the table is created in the database by connecting (`bin/psql-dev`), checking that it's there (`\dt`), and looking at the column value types and options (`\d form1299s;`) :

screen shot 2018-01-30 at 10 12 12 am

OR, 

3. Verify tests pass or the server runs: `make server_test` || `make server_run`

### CODE REVIEW

File: migrations/20180126235545_create_form1299s.up.fizz [hunk 1]
```diff
@@ -0,0 +1,57 @@
+create_table("form1299s", func(t) {
```
Comment (aileendds) [inline migrations/20180126235545_create_form1299s.up.fizz]: hello! are you planning on adding a service member ID field later?

Comment (tinyels) [reply migrations/20180126235545_create_form1299s.up.fizz]: That's a good question! At the moment we have been focusing on what is on the paper form. We will have a future story to address service member id.

Comment (tinyels) [reply migrations/20180126235545_create_form1299s.up.fizz]: added placeholder story for that: https://www.pivotaltracker.com/story/show/154782797

File: swagger.yaml [hunk 2]
```diff
@@ -143,94 +143,104 @@ definitions:
       date_prepared:
         type: string
         format: date
-        example: 2018-01-03
+        example: '2018-01-03'
```
Comment (macrael) [inline swagger.yaml]: Why do these all need quotes? Everything is a string by default in yaml

File: swagger.yaml [hunk 3]
```diff
@@ -143,94 +143,104 @@ definitions:
...
+        example: '555-55-5555'
         x-nullable: true
       service_member_agency:
         type: string
-        example: Air Force
+        example: 'Air Force'
         x-nullable: true
       hhg_total_pounds:
         type: integer
+        format: int64
```
Comment (macrael) [inline swagger.yaml]: I don't think we need to specify int64 in swagger.yaml. The docs say it is a hint for tooling but go-swagger already uses int64s so we can leave this out.

File: swagger.yaml (hunk 2 above)
Comment (amitch23) [reply swagger.yaml]: Oh ok. So the `-` distinguishes this date string as a string and not integer? I'll change it back then.

File: swagger.yaml (hunk 3 above)
Comment (amitch23) [reply swagger.yaml]: Got it, removed.

**Pull request merged**